project/AppDependencies.scala

```diff
@@ -5,28 +5,28 @@ import sbt._
 object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.18.0",
-    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.18.0",
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "0.68.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "1.22.0-play-28",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.12.3"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.1.0",
+    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "7.1.0",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28" % "0.71.0",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.23.0-play-28",
```

Comment:
Is there any reason for not updating to [3.23.0-play-28](https://github.com/hmrc/play-frontend-hmrc/releases/tag/v3.23.0)?

Reply:
please ignore